src/lib/journey/journey.store.ts

```diff
@@ -84,52 +69,72 @@ export async function initialize(journey?: string): Promise<InitObject> {
     const previousPayload = prevStep?.payload;
     let nextStep: StepTypes;
 
-    step.set(prevStep);
+    set({
+      completed: false,
+      error: null,
+      loading: true,
+      step: prevStep,
+      successful: false,
+      response: null,
+    });
+
     try {
       /**
        * Initial attempt to retrieve next step
        */
-      nextStep = await FRAuth.next(get(step as Writable<FRStep>), options);
+      nextStep = await FRAuth.next(prevStep as FRStep, options);
     } catch (err) {
       console.error(`Next step request | ${err}`);
 
       /**
        * Setup an object to display failure message
        */
       nextStep = new FRLoginFailure({
-        message: 'Unknown request failure'
+        message: 'Unknown request failure',
       });
     }
 
     if (nextStep.type === StepType.Step) {
-      step.set(nextStep);
-      submittingForm.set(false);
+      // Iterate on a successful progression
+      stepNumber = stepNumber + 1;
```

Comment:
This is just for tracking when we can autosubmit the form if the form data got stale. Step #1 is the only valid number to resubmit.